apps/vaporgui/VariablesWidget.cpp

```diff
@@ -370,6 +371,10 @@ string VariablesWidget::updateVarCombo(
 void VariablesWidget::updateScalarCombo() {
 	if (_variableFlags & SCALAR) {
 		string setVarReq = _rParams->GetVariableName();
+        
+        if (_dataMgr->VariableExists(_rParams->GetCurrentTimestep(), setVarReq))
+            _activeDim = _dataMgr->GetNumDimensions(setVarReq);
+        
```

Comment:
I believe that these four lines will have the correct behavior we put them in VariablesWidget::updateDimCombo().  I don't think this is the right place.